Gemfile

```diff
@@ -12,6 +12,8 @@ gem "turbo-rails"
 gem "importmap-rails"
 gem "standard"
 gem "appraisal"
+gem "i18n-tasks"
+gem "highline", "~> 2.0"
```

Comment:
this is one of i18n-tasks deps. I had to specify the version under 3.0 as 3.0 is ruby 3.0 onwards.